frontend/coprs_frontend/coprs/forms.py

```diff
@@ -1416,6 +1416,48 @@ def __init__(self, api_key=None, repo_url=None, *args, **kwargs):
             self.repo_url.data = repo_url
 
 
+class AbstractSeparatedListField(wtforms.Field):
```

Comment:
Note: Going with abstract because re-using this for the space separated fields seems reasonable in the future.